lib/decorators/deadline.js

```diff
@@ -0,0 +1,34 @@
+const { get } = require('lodash');
+const moment = require('moment-business-time');
+const Case = require('@ukhomeoffice/taskflow/lib/models/case');
```

Comment:
Is there a better way to get access to the Case model? This seems to work without requiring any further connection setup, but should we be passing it in instead?

Reply:
There _should_ be a better way. This way is bad. 

Let me have a look.

Reply:
Looks like there's already 4 other instances of this in the project, so if I wanted to have feelings about it then it's too late.

Crack on!